kernel/banner.c

```diff
@@ -16,11 +16,13 @@
 #define BANNER_POSTFIX ""
 #endif
 
+#ifndef BANNER_VERSION
```

Comment:
Why not add Kconfig for this, like `config BOOT_BANNER_VERSION`?

Reply:
Because a Kconfig would remain set and stale, if you were to update the base zephyr version for example but you configured your app on zephyr 3.1, despite having moving to zephyr 3.4, the Kconfig would still be set to version 3.1 in the configured project